<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QStarDict::MainWindow</class>
 <widget class="QMainWindow" name="QStarDict::MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QStarDict</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pixmaps/pixmaps.qrc">
    <normaloff>:/icons/qstardict.png</normaloff>:/icons/qstardict.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout">
    <property name="margin">
     <number>9</number>
    </property>
    <property name="spacing">
     <number>6</number>
    </property>
    <item row="0" column="0">
     <layout class="QHBoxLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="searchClearButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>60</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Clear the search box</string>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
        <property name="icon">
         <iconset resource="pixmaps/pixmaps.qrc">
          <normaloff>:/icons/clear-right.png</normaloff>:/icons/clear-right.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>70</width>
          <height>60</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonIconOnly</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="searchBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>55</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="queryButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Fuzzy query</string>
        </property>
        <property name="text">
         <string>Search</string>
        </property>
        <property name="icon">
         <iconset resource="pixmaps/pixmaps.qrc">
          <normaloff>:/icons/search-filter.png</normaloff>:/icons/search-filter.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="DictWidget" name="translationView" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>742</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menu_Options">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="actionScan"/>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Options"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QDockWidget" name="wordsListDock">
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::NoDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Words list</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout">
     <property name="margin">
      <number>9</number>
     </property>
     <property name="spacing">
      <number>6</number>
     </property>
     <item row="0" column="0">
      <widget class="QListWidget" name="wordsList">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="pixmaps/pixmaps.qrc">
     <normaloff>:/icons/application-exit.png</normaloff>:/icons/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="pixmaps/pixmaps.qrc">
     <normaloff>:/icons/configure.png</normaloff>:/icons/configure.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Configure</string>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
  </action>
  <action name="actionScan">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Scan</string>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="pixmaps/pixmaps.qrc">
     <normaloff>:/icons/help-contents.png</normaloff>:/icons/help-contents.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DictWidget</class>
   <extends>QWidget</extends>
   <header>dictwidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>searchBox</tabstop>
  <tabstop>queryButton</tabstop>
  <tabstop>wordsList</tabstop>
  <tabstop>searchClearButton</tabstop>
 </tabstops>
 <resources>
  <include location="pixmaps/pixmaps.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>searchClearButton</sender>
   <signal>clicked()</signal>
   <receiver>searchBox</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>647</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchClearButton</sender>
   <signal>clicked()</signal>
   <receiver>searchBox</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>647</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchBox</sender>
   <signal>returnPressed()</signal>
   <receiver>queryButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>438</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>661</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
